notebooks/official/tensorboard/tensorboard_hyperparameter_tuning_with_hparams.ipynb

```diff
@@ -685,23 +562,25 @@
+        "\n",
         "Click the generated TensorBoard link and click on \"HParams\" at the top.\n",
         "\n",
-        "The left pane of the dashboard provides filtering capabilities that are active across all the views in the HParams dashboard:\n",
+        "The left pane of the dashboard provides filtering capabilities that are active across all the views in the HParams dashboard. In this pane, you can:\n",
         "\n",
-        "- Filter which hyperparameters/metrics are shown in the dashboard\n",
-        "- Filter which hyperparameter/metrics values are shown in the dashboard\n",
-        "- Filter on run status (running, success, ...)\n",
-        "- Sort by hyperparameter/metric in the table view\n",
-        "- Number of session groups to show (useful for performance when there are many experiments)\n",
+        "- Filter which hyperparameters/metrics are shown in the dashboard.\n",
+        "- Filter which hyperparameter/metrics values are shown in the dashboard.\n",
+        "- Filter on run status (running, success etc.).\n",
+        "- Sort by hyperparameter/metric in the table view.\n",
+        "- Select number of session groups to show (useful for performance when there are many experiments).\n",
         "\n",
         "The HParams dashboard has three different views, with various useful information:\n",
         "\n",
         "* The **Table View** lists the runs, their hyperparameters, and their metrics.\n",
-        "* The **Parallel Coordinates View** shows each run as a line going through an axis for each hyperparemeter and metric. Click and drag the mouse on any axis to mark a region which will highlight only the runs that pass through it. This can be useful for identifying which groups of hyperparameters are most important. The axes themselves can be re-ordered by dragging them.\n",
+        "* The **Parallel Coordinates View** shows each run as a line going through an axis for each hyperparemeter and metric. Click and drag the mouse on any axis to mark a region which highlights only the runs that pass through it. This can be useful for identifying which groups of hyperparameters are most important. The axes themselves can be re-ordered by dragging them.\n",
         "* The **Scatter Plot View** shows plots comparing each hyperparameter/metric with each metric. This can help identify correlations. Click and drag to select a region in a specific plot and highlight those sessions across the other plots.\n",
         "\n",
-        "A table row, a parallel coordinates line, and a scatter plot market can be clicked to see a plot of the metrics as a function of training steps for that session (although in this tutorial only one step is used for each run)."
+        "A table view, a parallel coordinates view, and a scatter plot matrix view are also provided. These views help to see the plots of the metrics as a function of training steps for that session (although in this tutorial only one step is used for each run)."
```

Comment:
- change:  "A table view, a parallel coordinates view, and a scatter plot matrix view are also provided."
- to: "Also provided are a table view, a parallel coordinates view, and a scatter plot matrix view."

Reply:
This sentence seems unnecessary. The bullet point descriptions already cover these views. Hence, removing the sentence.